File: OLD/SUNBEAM CDAC/cpp/Day10/demo04/person.cpp
#include "person.h"

Person::Person()
{
    cout << "Person Parameterless" << endl;
}

Person::Person(string name)
{
    cout << "Person Parameterized" << endl;
}
void Person::accept()
{
    cout << "Person acccept" << endl;
}
void Person::display()
{
    cout << "Person display" << endl;
}

File: OLD/SUNBEAM CDAC/java/Day05/Demo04/src/com/sunbeam/Person.java
package com.sunbeam;

import java.util.Scanner;

public class Person {
	private String name;
	private String mobile;
	
	public Person() {
		System.out.println("Person()");
	}
	
	public Person(String name, String mobile) {
		System.out.println("Person(String, String)");
		this.name = name;
		this.mobile = mobile;
	}

	public void accept(Scanner sc) {
		System.out.println("Enter name -");
		name = sc.next();
		System.out.println("Enter mobile -");
		mobile = sc.next();
		
	}
	
	public void display() {
		System.out.println("Name - "+name);
		System.out.println("Mobile - "+mobile);
	}
}
